app.py

```diff
@@ -81,7 +82,7 @@ def check_for_admins():
     )
     assert (os.path.isfile('./data.db')), errorMsg
     try:
-        admins = UserModel.find_by_role('admin')
+        admins = UserModel.find_by_role(4)
```

Comment:
You're right, I should have imported the enum from the user model. Do you think we should store the value in the database as an enum type instead of int? I think we last decided a simple int would suffice that gets evaluated against the enum.

Reply:
You're right, I should have imported the enum from the user model. Do you think we should store the value in the database as an enum type instead of int? I think we last decided a simple int would suffice that gets evaluated against the enum.

Reply:
That is a good question, a question I would answer differently depending on the situation. I have no strong opinions one way or another. In this case I would probably prefer to use an enum in the database, this would have the added benefit of having a default role (Pending) added automatically by the db to all newly created users. It turns out we are currently not using the defined UserRole enum anywhere. So if you want to change it then now would be a good time.

Reply:
I do agree we should be using the UserRole enum as much as possible